orc/src/main/java/org/apache/iceberg/orc/ORC.java

```diff
@@ -86,9 +85,6 @@
 @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
 public class ORC {
 
-  /** @deprecated use {@link TableProperties#ORC_WRITE_BATCH_SIZE} instead */
-  @Deprecated private static final String VECTOR_ROW_BATCH_SIZE = "iceberg.orc.vectorbatch.size";
```

Comment:
While we don't want people using this, does it make sense to ever actually remove it? It could cause behavior changes that we don't really need to.